### ISSUE TITLE
Offer hosting services 

### ISSUE THREAD

**Issue by nobaraos12** [JulianPrieber/littlelink-custom#228]
Offer hosting services such as tiers (free, paid, business)

**Comment by JulianPrieber** [comment]
Like [this](https://www.patreon.com/JulianPrieber)?

**Comment by nobaraos12** [comment]
yes but display it on the man page your services, will only help fund the project

**Comment by JulianPrieber** [comment]
Like [this](https://littlelink-custom.com/sponsor)?

**Comment by nobaraos12** [comment]
that looks good, why not add it to the header as an "Pricing" category then you can also add the free/demo tier. I was not able to find your sponsors page.
